unstructured/ingest/v2/examples/example_pinecone.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+
+from unstructured.ingest.v2.interfaces import ProcessorConfig
+from unstructured.ingest.v2.logger import logger
+from unstructured.ingest.v2.pipeline.pipeline import Pipeline
+from unstructured.ingest.v2.processes.chunker import ChunkerConfig
+from unstructured.ingest.v2.processes.connectors.local import (
+    LocalConnectionConfig,
+    LocalDownloaderConfig,
+    LocalIndexerConfig,
+)
+from unstructured.ingest.v2.processes.connectors.pinecone import (
+    PineconeAccessConfig,
+    PineconeConnectionConfig,
+    PineconeUploaderConfig,
+    PineconeUploadStagerConfig,
+)
+from unstructured.ingest.v2.processes.embedder import EmbedderConfig
+from unstructured.ingest.v2.processes.partitioner import PartitionerConfig
+
+base_path = Path(__file__).parent.parent.parent.parent.parent
+docs_path = base_path / "example-docs"
+work_dir = base_path / "tmp_ingest"
+output_path = work_dir / "output"
+download_path = work_dir / "download"
+
+if __name__ == "__main__":
+    logger.info(f"Writing all content in: {work_dir.resolve()}")
+    Pipeline.from_configs(
+        context=ProcessorConfig(work_dir=str(work_dir.resolve())),
+        indexer_config=LocalIndexerConfig(
+            input_path=str(docs_path.resolve()) + "/book-war-and-peace-1p.txt"
+        ),
+        downloader_config=LocalDownloaderConfig(download_dir=download_path),
+        source_connection_config=LocalConnectionConfig(),
+        partitioner_config=PartitionerConfig(strategy="fast"),
+        chunker_config=ChunkerConfig(chunking_strategy="by_title"),
+        embedder_config=EmbedderConfig(embedding_provider="langchain-huggingface"),
+        destination_connection_config=PineconeConnectionConfig(
+            # You'll need to set PINECONE_API_KEY environment variable to run this example
+            access_config=PineconeAccessConfig(api_key=os.getenv("PINECONE_API_KEY")),
+            index_name="your index name here. e.g. my-index",
```

Comment:
use more `os.getenv()` calls for these.

Reply:
[use more os getenvs in the example](https://github.com/Unstructured-IO/unstructured/pull/3286/commits/de14f299d7abba9f7857f6dc482028c8367dade3)